package.json

```
@@ -5,12 +5,18 @@
   "main": "app.js",
   "scripts": {
     "dev": "nodemon app.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "echo \"Error: no test specified\" && exit 1"
```

```suggestion
    "start": "node app.js"
```
What! jajaja cuando ejecutas start quieres que salga un mensaje de Error... o quieres arrancar la app? xD